python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py

```diff
@@ -249,6 +250,27 @@ class Meta:
         name = "TerminateRunResult"
 
 
+class GrapheneTerminateRunsResult(graphene.ObjectType):
+    """Indicates the runs that successfully terminated and those that failed to terminate."""
+
+    runIdsTerminated = non_null_list(graphene.String)
+    runIdsFailedToTerminate = non_null_list(graphene.String)
```

Comment:
this could also be a list of GrapheneTerminateRunFailure and a list of GrapheneTerminateRunSuccess? that way it would include the failure message if there is one

Reply:
I'll return a list of `GrapheneTerminateRunFailure`s, agree the error message would be helpful.

I think we can continue returning the run ids successfully terminated though, `GrapheneTerminateRunSuccess` returns the whole run object which could be a lot for tens of thousands of runs

Reply:
it's graphql so it only returns the fields they request right?

i.e. you could do 
```
{
   ... on TerminateRunSuccess {
     run {
       id
     }
   }
}
```

and only get the run IDs back

Reply:
ah right good call